src/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -48,13 +48,39 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = "Reads a SAM or BAM file and writes a file containing metrics about " +
-                "the statistical distribution of insert size (excluding duplicates) " +
-                "and generates a Histogram plot.",
-        usageShort = "Writes insert size distribution metrics for a SAM or BAM file",
+        usage = CollectInsertSizeMetrics.USAGE_SUMMARY,
+        usageShort = CollectInsertSizeMetrics.USAGE_BRIEF,
         programGroup = Metrics.class
 )
 public class CollectInsertSizeMetrics extends SinglePassSamProgram {
+    static final String USAGE_BRIEF = "Metrics about the insert size distribution of a paired-end library";
+    static final String USAGE_SUMMARY = "Metrics about the insert size distribution of a paired-end library, created by the " +
+            "CollectInsertSizeMetrics program and usually written to a file with the extension " +
+            "\".insert_size_metrics.txt\".  In addition the insert size distribution is plotted to a file with the " +
+            "extension \".insert_size_Histogram.pdf\".<br /><br />" +
+            "There are two separate outputs from this tool including distributions of insert sizes as well as " +
```

Comment:
I prefer a more active voice for these descriptions, i.e. 

> There are two separate outputs from this tool (...)

should be more like 

> This tool produces two separate outputs (...)